ibis/tests/all/conftest.py

```diff
@@ -16,8 +16,8 @@
     pytest.param(Postgres, marks=pytest.mark.postgres),
     pytest.param(MySQL, marks=pytest.mark.mysql),
     pytest.param(Clickhouse, marks=pytest.mark.clickhouse),
+    pytest.param(Impala, marks=pytest.mark.impala),
```

Comment:
If we're moving things around here let's sort by backend name.

Reply:
Just wanted to put bigquery to the end - due to bq is the slowest one.

Reply:
Ah ok. Then that's fine, let's leave it.